sdk/nodejs/automation/cmd.ts

```diff
@@ -13,9 +13,18 @@
 // limitations under the License.
 
 import execa from "execa";
-
+import * as fs from "fs";
+import got from "got";
+import * as os from "os";
+import * as path from "path";
+import * as semver from "semver";
+import * as tmp from "tmp";
+import { version as DEFAULT_VERSION } from "../version";
+import { minimumVersion } from "./minimumVersion";
 import { createCommandError } from "./errors";
 
+const SKIP_VERSION_CHECK_VAR = "PULUMI_AUTOMATION_API_SKIP_VERSION_CHECK";
```

Comment:
We should add this envvar to sdk/go/common/env/env.go

Reply:
Since this is the nodejs SDK, should I move the [go version of the env var](https://github.com/pulumi/pulumi/blob/097fdaa09a1a8b6c76b6aab877fa6ffc92c43369/sdk/go/auto/local_workspace.go#L63) in the PR for go?

Reply:
Yeh as long as we move it at some point, because then it will show in `pulumi env` help.